<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HAdjustEditDialog</class>
 <widget class="QDialog" name="HAdjustEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>调整数据编辑</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="3">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>15</number>
     </property>
     <widget class="QGroupBox" name="groupBox_1">
      <property name="title">
       <string>相对光谱能量分布</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QSplitter" name="splitter_1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>15</number>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>调整数据</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>测试参数设置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="0"/>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
